frontend/public/module/k8s/selector.ts

```diff
@@ -29,10 +34,9 @@ export const fromRequirements = (requirements, options) => {
   return selector;
 };
 
-export const split = string => string.trim() ? string.split(/,(?![^(]*\))/) : []; // [''] -> []
+export const split = (string: string) => string.trim() ? string.split(/,(?![^(]*\))/) : []; // [''] -> []
```

Comment:
not new, but might be good to call the vars here and below `str` to avoid confusion with TypeScript `string`